packages/esm-bed-management-app/src/bed-type/bed-type-administration-table.component.tsx

```diff
@@ -82,21 +117,35 @@ const BedTypeAdministrationTable: React.FC = () => {
         displayName: entry?.displayName,
         description: entry?.description,
         actions: (
-          <IconButton
-            align="top-start"
-            enterDelayMs={300}
-            kind="ghost"
-            label={t('editBedType', 'Edit bed type')}
-            onClick={(e) => {
-              e.preventDefault();
-              openEditBedTypeModal(entry);
-            }}
-            size={responsiveSize}>
-            <Edit />
-          </IconButton>
+          <>
+            <IconButton
+              align="top-start"
```

Comment:
Lately I've been in favour of using the exprerimental `autoAlign` value for this prop.

Reply:
Are we already using this prop elsewhere? A bit concerned since its experimental

Reply:
There's some occurrences in the Stock management app https://github.com/search?q=org:openmrs%20autoAlign&type=code